docs/application/dotnet/get-started/nui/quickstart.md

```diff
@@ -0,0 +1,288 @@
+The Tizen .NET framework allows you to easily and efficiently create applications for Tizen. Study the following instructions to help familiarize yourself with the Natural User Interface (NUI) application development process. With the instructions, you can create and run a basic NUI application, which displays some text on the screen with no user interaction.
+
+   > **Note**
+   >
+   > .NET NUI applications are supported since Tizen 4.0
+
+1. Before you get started with developing Tizen applications, set up the [development environment](../../../vstools/install.md).
+
+2. [Creating a Project](#create-a-project) using Visual Studio.
+  This step shows how you can use a predesigned project template that
```

Comment:
From line 9 to line 18.
To get the line indentation, please add one more step to each sub lines.
For now these lines are all merge into one line.
![image](https://user-images.githubusercontent.com/9958374/68914280-73abf600-07a1-11ea-86fc-915384a5b256.png)